docs/publish/publish-add-in-vs-code.md

```diff
@@ -29,9 +29,9 @@ When your add-in is working as desired, you can publish it directly through Visu
 2. From the Extensions view in VS Code, search for the Azure Storage extension and install it.
```

Comment:
Nit: Do you want to adjust this numbered list so all the steps use "1." and the numbering is done automatically?